### ISSUE TITLE
Dataset Issues

### ISSUE THREAD

**Issue by Kunind27** [gospodima/Extended-SimGNN#6]
I checked out the dataset already uploaded onto PyTorch Geometric (the GEDDataset Class).
I don't know about you, but the problem I faced was that for all the GEDDatasets - Alkane, Linux and AIDS700nef, the GEDs for a pair of Graphs G1, G2 where both G1 and G2 belong to the TEST SET is set to 'inf'
How did you deal with this?
Also the feature matrix for the nodes (i.e data.x) is only given for the AIDS700nef dataset and not the other two - so how did you deal with that

**Comment by gospodima** [comment]
Hello @Kunind27,

> the GEDs for a pair of Graphs G1, G2 where both G1 and G2 belong to the TEST SET is set to 'inf'

In my implementation I used train-train graph pairs for training and train-test graph pairs for evaluation. 

> Also the feature matrix for the nodes (i.e data.x) is only given for the AIDS700nef dataset and not the other two - so how did you deal with that

I used [OneHotDegree](https://pytorch-geometric.readthedocs.io/en/latest/_modules/torch_geometric/transforms/one_hot_degree.html). Refer the [code](https://github.com/gospodima/Extended-SimGNN/blob/9b871284b2553a59dd1e1f40a4d10de39667ebc8/src/simgnn.py#L221).
